tests/test_config.py

```diff
@@ -349,7 +349,10 @@ def ld_i8(
     ld_i8 = ld_i8.configwrite_after('ConfigLoad.scale = _', ConfigLoad,
                                     'src_stride', 'stride(src, 0)')
     ld_i8 = ld_i8.replace(do_ld_i8, 'for i in _:_')
-    ld_i8 = ld_i8.replace(config_ld_i8, 'ConfigLoad.scale = scale')
+    ld_i8 = ld_i8.replace(
```

Comment:
Do we have a mechanism to specify how many statements we want the selection cursor to hold? For example, it'll be nice to write here `ld_i8.find_cursor('ConfigLoad.scale = _;', 2)` or sth like that.

Reply:
Every cursor in the list returned by this call will be a 1-statement selection. However, the list might be of length greater than 1 if there are multiple matches. We might want to add a method to `Selection` to _extend_ the selection to include the next `n` statements. With what's here now you could write:

```
[sel[:2] for sel in ld_i8.find_cursor('ConfigLoad.scale = _ ; _')]
```

which would ensure each match had length at _most_ 2.

Reply:
You could also imagine extending the pattern language with something like:

```
ld_i8.find_cursor('ConfigLoad.scale = _ ; _[1]')
```
where `_[1]` means "match exactly 1 statement". You could also do things like `_[:2]` to mean "match up to two" or `_[3:]` to mean "match at least three".

Reply:
I think your first idea is clearer and makes more sense, extending the string pattern matching doesn't sound like a good idea.

Reply:
A Window expression and a selection are entirely different concepts... I'm not sure how they would relate. The indexing operations Selection supports are basically the same as those of a Python list, you just can't step by anything other than 1.

Reply:
Does `Selection` support range (e.g., `s[1:3]`)?

Reply:
Yes, that statement returns a sub-selection containing the second and third statements of `s`, the same as a Python list.